index.js

```diff
@@ -0,0 +1,30 @@
+// const express = require('express')
```

Comment:
Add a README.md file to the repo and add documentation on what the module does and how to use it. Move this example to that file.